sites-available/example.com.conf

```diff
@@ -38,12 +38,6 @@ server {
         return 444;
     }
 
-    ## Protection against illegal HTTP methods. Out of the box only HEAD,
-    ## GET and POST are allowed.
-    if ($not_allowed_method) {
-        return 405;
-    }
-
```

Comment:
You're removing this from the default because you are assuming people have dgo.to/js installed but not everyone will, so my opinion is that it should be all commented out and let users enable it on their own.